app/src/main/java/org/wikipedia/analytics/metricsplatform/MetricsPlatform.kt

```diff
@@ -3,22 +3,37 @@ package org.wikipedia.analytics.metricsplatform
 import org.wikimedia.metrics_platform.MetricsClient
 import org.wikimedia.metrics_platform.context.AgentData
 import org.wikimedia.metrics_platform.context.ClientData
+import org.wikimedia.metrics_platform.context.MediawikiData
 import org.wikipedia.WikipediaApp
 import org.wikipedia.settings.Prefs
+import org.wikipedia.util.ReleaseUtil
 import java.time.Duration
 
 object MetricsPlatform {
-    private val agentData = AgentData(
+    val agentData = AgentData(
         WikipediaApp.instance.appInstallID,
         "android",
         "app"
     )
 
+    val mediawikiData = MediawikiData(
+        WikipediaApp.instance.currentTheme.toString(),
+        WikipediaApp.instance.versionCode.toString(),
+        ReleaseUtil.isProdRelease,
+        ReleaseUtil.isDevRelease,
+        WikipediaApp.instance.wikiSite.dbName(),
+        WikipediaApp.instance.languageState.systemLanguageCode,
+        null
+    )
+
+    val domain = WikipediaApp.instance.wikiSite.authority()
+
     private val clientData = ClientData(
         agentData,
         null,
+        mediawikiData,
         null,
-        WikipediaApp.instance.wikiSite.authority()
+        domain
```

Comment:
I know it's weird to have client data initiated with the MP client and then passed in with the event -- there's likelihood that the MP monoschema will change here soon so rather than revamping the functionality now, I err'd on the side of getting this out the door to start collecting data to help inform how the monoschema should change